consensus/polybft/consensus_runtime_test.go

```diff
@@ -1982,34 +1982,71 @@ func TestConsensusRuntime_HasQuorum(t *testing.T) {
 	}
 
 	require.NoError(t, runtime.FSM())
+	proposer, err := runtime.fsm.validators.CalcProposer(0)
+
+	require.NoError(t, err)
 
 	messages := make([]*proto.Message, 0, len(validatorAccounts.validators))
 
-	for _, x := range validatorAccounts.validators {
-		messages = append(messages, &proto.Message{
-			From: x.Address().Bytes(),
-		})
-	}
+	// Unknown message type
+	assert.False(t, runtime.HasQuorum(lastBuildBlock.Number+1, messages, -1))
 
 	// invalid block number
 	for _, msgType := range []proto.MessageType{proto.MessageType_PREPREPARE, proto.MessageType_PREPARE,
 		proto.MessageType_ROUND_CHANGE, proto.MessageType_COMMIT} {
 		assert.False(t, runtime.HasQuorum(lastBuildBlock.Number, nil, msgType))
 	}
 
-	// Unknown message type
-	assert.False(t, runtime.HasQuorum(lastBuildBlock.Number+1, messages, -1))
-
 	// MessageType_PREPREPARE
+	for _, x := range validatorAccounts.validators {
+		messages = append(messages, &proto.Message{
+			From: x.Address().Bytes(),
+			Type: proto.MessageType_PREPREPARE,
+		})
+	}
```

Comment:
Why not having only single `pre-prepare` message, sent only from the proposer?

Reply:
fixed https://github.com/0xPolygon/polygon-edge/commit/5eb6d876a90a3be1d8311f75ab4947f44be04baa